submissions/zalevskyi/dom/js/app.js

```diff
@@ -0,0 +1,142 @@
+        description: 'Shrewd and fiscally knowledgeable, Reyna is able to strengthen the economic power of a city, using finances to provide more than is normally possible.',
+        context: 'A financier is an official tasked with resolving budgetary and financial matters, requiring an astute understanding of economics and the psychology of markets. Playing with the money of the state is not a matter for the faint-hearted.',
+        img_src: './img/Reyna.png',
+        img_alt: 'Reyna portrait'
+    },
+    'victor' : {
+        type: 'governor',
+        title: 'The Castellan',
+        description: 'Military focused, once established in a city, Victor is instrumental in strengthening, preserving, and defending it from foreign threats.',
+        context: 'A Castellan is a military officer responsible for directing the day-to-day affairs of a castle or fortress and the territory immediately around it. As castles were massive investments of treasure and important points of military power, the castellan had to be both competent and politically reliable. In times of war, a competent castellan could determine the fate of campaigns, and with that, the historical outcome.',
+        img_src: './img/Victor.png',
+        img_alt: 'Victor portrait'
+    }
+}
+Object.freeze(WIKI)
+const MAIN_INNER_HTML = `<input type='button' id='menu_button' class='menu_button' value='Menu'>
+                         <div id='civilopedia_path' class='civilopedia_path'>Civilization VI | Governors</div>
+                         <div id='article' class='article'><h1 id='title' class='text_heading_1'></h1>
+                            <img id='portrait' alt='' src='' class='portrait'>
+                            <h2 class='text_heading_2 description_heading'>Description</h2>
+                            <p id='description' class='text description'></p>
+                            <h2 class='text_heading_2 context_heading'>Historical Context</h2>
+                            <p id='context' class='text context'></p>
+                         </div>`
+const MAIN = document.getElementById('main')
+const NAV = document.getElementById('nav')
+MAIN.innerHTML=MAIN_INNER_HTML
+const TITLE = document.getElementById('title')
+const DESCRIPTION = document.getElementById('description')
+const PORTRAIT = document.getElementById('portrait')
+const CONTEXT = document.getElementById('context')
+const GOVERNOR_HEADINGS = document.querySelectorAll('.text_heading_2')
+let menu_item_current
+
+function createMenu() {
+    let navHTML = `<ul id='menu' class='menu_list'>`
+    navHTML += Object.keys(WIKI).map(element => `<li><a href='#' class='menu_item'>${element}</a></li>`).join('')
+    navHTML +=`</ul>`
+    NAV.innerHTML = navHTML
+    const MENU_ITEMS = document.querySelectorAll('.menu_item')
+    MENU_ITEMS[0].classList.add('menu_item_first')
+    MENU_ITEMS[MENU_ITEMS.length-1].classList.add('menu_item_last')
+    return MENU_ITEMS[0]
+}
+function selectMenuItem(item) {
+    try {
```

Comment:
I'd recommend you to get rid of this try-catch construction here.
No need to store this variable as global because it can be not so easy to check where it was changed etc.
You can pass the `target` to the `updateMain` function and work with it there.
Maybe my suggestion would help you to make this function more clean:
```suggestion
    function selectMenuItem(target) {
       target.classList.remove('menu_item_selected');
       target.classList.add('menu_item_selected');
       updateMain(target);
    }
```